conf/cassandra.yaml

```diff
@@ -2368,3 +2368,111 @@ drop_compact_storage_enabled: false
+#       # clusters with 5 nodes that finish repairs quickly. The default is 24 hours. This means that if the scheduler
+#       # completes one round on all nodes in less than 24 hours, it will not start a new repair round on a given node
+#       # until 24 hours have passed since the last repair.
+#       min_repair_interval: 24h
+#       token_range_splitter:
+#         parameters:
+#           # The target and maximum amount of bytes that should be included in a repair
+#           # assignment. This scopes the amount of work involved in a repair and includes
+#           # the data covering the range being repaired.
+#           bytes_per_assignment: 200GiB
+#           # The maximum number of bytes to cover in an individual schedule. This serves as
+#           # a mechanism to throttle the work done in each repair cycle. You may reduce this
+#           # value if the impact of repairs is causing too much load on the cluster or increase it
+#           # if writes outpace the amount of data being repaired. Alternatively, adjust the
+#           # min_repair_interval.
+#           max_bytes_per_schedule: 100000GiB
+#     incremental:
+#       enabled: true
+#       # Incremental repairs finish quickly, so they can be run more frequently. The default is 1 hour.
+#       min_repair_interval: 1h
+#       token_range_splitter:
+#         parameters:
+#           # Configured to attempt repairing 50GiB of data per repair.
+#           # This throttles the amount of incremental repair and anticompaction done per schedule
+#           # after incremental repairs are turned on.
+#           bytes_per_assignment: 50GiB
+#           # The maximum number of bytes to cover in an individual schedule to 100GiB.
+#           # Consider increasing max_bytes_per_schedule if
+#           # more data is written than this limit within the min_repair_interval.
+#           max_bytes_per_schedule: 100GiB
+#     preview_repaired:
+#       enabled: true
+#       min_repair_interval: 24h
+#       token_range_splitter:
+#         parameters:
+#           bytes_per_assignment: 200GiB
+#           max_bytes_per_schedule: 100000GiB
+#   # Time interval between successive checks to see if ongoing repairs are complete or if it is time to schedule
+#   # repairs.
+#   repair_check_interval: 5m
+#   # Maximum number of retries for a repair session.
+#   repair_max_retries: 3
+#   # Backoff time before retrying a repair session.
+#   repair_retry_backoff: 30s
+#   # Minimum duration for the execution of a single repair task. This prevents the scheduler from overwhelming
+#   # the node by scheduling too many repair tasks in a short period of time.
+#   repair_task_min_duration: 5s
+#   # The scheduler needs to adjust its order when nodes leave the ring. Deleted hosts are tracked in metadata
+#   # for a specified duration to ensure they are indeed removed before adjustments are made to the schedule.
+#   history_clear_delete_hosts_buffer_interval: 2h
+#   global_settings:
```

Comment:
To make it more clean, should add a note that each of these settings can be overridden per repair type under repair_type_overrides.